### ISSUE TITLE
(error) Failed Innertube request when trying to access my playlists

### ISSUE THREAD

**Issue by lovebug0706** [Rehike/Rehike#559]
I don't know why, but i keep getting this error message each time i try to access my playlists folder. I can still enter playlists and see whats inside it, like my liked videos or someone else's playlist, but when I try to load up all of my playlists, it refuses. Here's the log:
```
 Failed request information

    Response status: 400
    Response content:
    { "error": { "code": 400, "message": "Request contains an invalid argument.", "errors": [ { "message": "Request contains an invalid argument.", "domain": "global", "reason": "badRequest" } ], "status": "INVALID_ARGUMENT" } }
    Response headers:
        vary: Array
        content-type: application/json; charset=UTF-8
        content-encoding: gzip
        date: Fri, 12 Apr 2024 02:08:07 GMT
        server: scaffolding on HTTPServer2
        cache-control: private
        content-length: 173
        x-xss-protection: 0
        x-frame-options: SAMEORIGIN
        x-content-type-options: nosniff
        set-cookie: Array
        alt-svc: h3=":443"; ma=2592000,h3-29=":443"; ma=2592000
```
```
Rehike\Exception\Network\InnertubeFailedRequestException: 
  at Rehike\Network::Rehike\{closure}(YukisCoffee\CoffeeRequest\Network\Response)(rehike://modules/Rehike/Network.php:176)
  at YukisCoffee\CoffeeRequest\Promise::resolve(YukisCoffee\CoffeeRequest\Network\Response)(rehike://modules/YukisCoffee/CoffeeRequest/Promise.php:285)
  at YukisCoffee\CoffeeRequest\Util\QueuedPromiseResolver::finish()(rehike://modules/YukisCoffee/CoffeeRequest/Util/QueuedPromiseResolver.php:57)
  at YukisCoffee\CoffeeRequest\Loop::finishQueuedPromises()(rehike://modules/YukisCoffee/CoffeeRequest/Loop.php:240)
  at YukisCoffee\CoffeeRequest\Loop::cleanup()(rehike://modules/YukisCoffee/CoffeeRequest/Loop.php:230)
  at YukisCoffee\CoffeeRequest\Loop::run()(rehike://modules/YukisCoffee/CoffeeRequest/Loop.php:84)
  at YukisCoffee\CoffeeRequest\CoffeeRequest::run()(rehike://modules/YukisCoffee/CoffeeRequest/CoffeeRequest.php:119)
  at Rehike\Network::run()(rehike://modules/Rehike/Network.php:373)
  at Rehike\Controller\core\HitchhikerController::get(Rehike\YtApp, "", Rehike\ControllerV2\RequestMetadata)(rehike://controllers/core/HitchhikerController.php:108)
  at Rehike\ControllerV2\GetControllerInstance::_cv2WrappedCallControllerMethod("get", array<0>)(rehike://modules/Rehike/ControllerV2/GetControllerInstance.php:51)
  at Rehike\ControllerV2\GetControllerInstance::__call("get", array<0>)(rehike://modules/Rehike/ControllerV2/GetControllerInstance.php:74)
  at Rehike\ControllerV2\Router::pointerHandler("feed", "get")(rehike://modules/Rehike/ControllerV2/Router.php:240)
  at Rehike\ControllerV2\Router::baseRequestMethod(array<30>, "get")(rehike://modules/Rehike/ControllerV2/Router.php:179)
  at Rehike\ControllerV2\Router::get(array<30>)(rehike://modules/Rehike/ControllerV2/Router.php:39)
  at require("C:\xampp\htdocs\router.php")(rehike://router.php:80)
  at Rehike\Boot\Bootloader::postboot()(rehike://modules/Rehike/Boot/Bootloader.php:88)
  at Rehike\Boot\Bootloader::startSession()(rehike://modules/Rehike/Boot/Bootloader.php:26)
  at {main}(rehike://index.php:71)
```

I have to exit out of Rehike for it to show my playlists, but when i return to Rehike when in one of my playlists, my playlist still works. Its just that i cant load all of my playlists through Rehike. I hope there's a fix soon, because this is one of my main things I use on YouTube. Thank you!

**Comment by kawapure** [comment]
This bug occurs because they moved playlists to a distinct feed page. I will work on patching this ASAP.

**Comment by kawapure** [comment]
It is fixed! 757922a15a7f6cdc2db8b59d1098d08eeee40776 restores the old behaviour, so your playlists should just show in the guide as they used to.
